app/src/main/java/com/vultisig/wallet/presenter/keygen/KeyGenErrorScreen.kt

```diff
@@ -0,0 +1,136 @@
+package com.vultisig.wallet.presenter.keygen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import androidx.navigation.compose.rememberNavController
+import com.vultisig.wallet.R
+import com.vultisig.wallet.app.ui.theme.appColor
+import com.vultisig.wallet.app.ui.theme.dimens
+import com.vultisig.wallet.app.ui.theme.menloFamily
+import com.vultisig.wallet.app.ui.theme.montserratFamily
+import com.vultisig.wallet.presenter.base_components.MultiColorButton
+import com.vultisig.wallet.presenter.common.TopBar
+
+@Composable
+fun KeyGenErrorScreen(
+    navController: NavController
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(MaterialTheme.appColor.oxfordBlue800)
+    ) {
+        Column(
+            modifier = Modifier.align(Alignment.TopCenter)
+        ) {
+            Spacer(modifier = Modifier.height(MaterialTheme.dimens.small2))
+
+            TopBar(
+                centerText = stringResource(R.string.keysign),
+                startIcon = null,
+                navController = rememberNavController()
+            )
+
+            Image(
```

Comment:
Could you please remove this image , we don't need it , even it is in figma , but this is keygen error

Reply:
ok